api/v1/qdrantcluster_types.go

```diff
@@ -181,9 +181,40 @@ func (s QdrantClusterSpec) GetServicePerNode() bool {
 }
 
 type GPU struct {
-	// GPUType specifies the type of the GPU to use.
+	// GPUType specifies the type of the GPU to use. If set, GPU indexing is enabled.
 	// +kubebuilder:validation:Enum=nvidia;amd
 	GPUType GPUType `json:"gpuType"`
+	// ForceHalfPrecision for `f32` values while indexing.
+	// `f16` conversion will take place
+	// only inside GPU memory and won't affect storage type.
+	// +kubebuilder:default=false
+	ForceHalfPrecision bool `json:"forceHalfPrecision"`
+	// DeviceFilter for GPU devices by hardware name. Case-insensitive.
+	// List of substrings to match against the gpu device name.
+	// Example: [- "nvidia"]
+	// Default: all devices are accepted.
```

Comment:
you mean if not specified all devices are accepted?

Reply:
I clarified the docs